e2e/output/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Do we need to reference .gitignore in .gitignore? :D

Reply:
It's an old trick to include an empty directory in git: https://stackoverflow.com/questions/115983/how-do-i-add-an-empty-directory-to-a-git-repository 

I have added an explaining comment to the file, good catch! :)

Reply:
Hm, wouldn't .gitkeep do the same here?

Reply:
Yes, although in this case we also want to ignore the content of the folder (while keeping the folder itself). We can do a combo of `.gitkeep` and `.gitignore` with `*` to make that intention clearer, if you prefer :)